docs/docsite/rst/shared_snippets/basic_concepts.txt

```diff
@@ -1,42 +1,55 @@
 Control node
 ============
 The machine from which you run the Ansible CLI tools (``ansible-playbook`` , ``ansible``, ``ansible-vault`` and others).
+
 You can use any computer that meets the software requirements as a control node - laptops, shared desktops, and servers can all run Ansible.
+
+You can also run Ansible in containers known as `execution envoronments <https://docs.ansible.com/ansible/devel/getting_started_ee/index.html>`_.
```

Comment:
Shouldn't this use a `:ref:`?

Reply:
I think because it's currently only on the `devel` branch it needs this format. We'll also need to update this line again if we change the location of the execution environments getting started.

Reply:
sure, let's do it later